main.tf

```diff
@@ -0,0 +1,101 @@
+locals {
```

Comment:
Both the root and module should have outputs.tf. They can be blank, but I'm guessing there's some info that would be good to print out.

Reply:
I assume we will figure this out as/after the various pieces get stitched together